leetcode/src/main/java/com/leet/SmallerNoArray.java

```diff
@@ -0,0 +1,47 @@
+package com.leet;
+
+// class name should be in camel case
+public class SmallerNoArray {
+    public static int[] smallerNumbersThanCurrent(int[] nums) {
+
+// write array declaration in java style (not in c style) -> int[] returnNums
+        // look for intellij shortcut to auto format code
+        int[] returnNums = new int[nums.length];
+        int count = 0;
+
+        for (int i = 0; i < nums.length; i++) {
+            for (int j = 0; j < nums.length; j++) {
+
+                if (j != i && nums[j] < nums[i]) {
+
+                    count++;
+                }
+
```

Comment:
why there are 2-3 lines between 2 lines of code. Try to write clean and formatted code.

Reply:
ok